src/methods/balances/transfer.spec.ts

```diff
@@ -17,4 +17,19 @@ describe('balances::transfer', () => {
       '0x060096074594cccf1cd185fa8a72ceaeefd86648f8d45514f3ce33c31bdd07e4655d30'
     );
   });
+
+  it('should accept big numbers as string', () => {
+    const unsigned = transfer(
+      {
+        ...TEST_METHOD_ARGS.balances.transfer,
+        value: '9007199254740996', // MAX_SAFE_INTEGER + 5
```

Comment:
@joepetrowski Do you have a repro for #132? I added this test, and it seems to work.

Reply:
I think that should be fine. I discovered it by trying to transfer 100k DOTs on a testnet, thought it might be the reason it failed, so I did 9007 and it worked and 9008 and it failed.